src/locales/zh.json

```diff
@@ -0,0 +1,98 @@
+{
+  "almost_there": "快完成了,",
+  "choose_carefully": "请仔细选择，选择后无法更改",
+  "choose_username": "选择用户名",
+  "confirm_account": "确认账号",
+  "confirm_existence": "我们需要确认是否存在",
+  "confirm_password": "请输入密码以确认",
+  "confirmation_code": "验证码",
+  "continue": "继续",
+  "copy": "复制",
+  "country_code_select": "选择地区",
+  "edit": "编辑",
+  "email": "E-mail",
+  "enjoy_steem": "你可以使用Steem所有功能了",
+  "enter_confirmation_code": "输入验证码",
+  "enter_email": "输入E-mail地址",
+  "enter_phone": "输入你的手机号码",
+  "error_api_account_created": "账号已存在",
+  "error_api_account_longer": "用户名应该更长",
+  "error_api_account_verification_pending": "账号还未被确认",
+  "error_api_code_invalid": "无效的验证码",
+  "error_api_code_required": "验证码是必填的",
+  "error_api_country_code_required": "地区是必选的",
+  "error_api_create_account": "创建账号时发生错误",
+  "error_api_domain_blacklisted": "此域名已被加入黑名单，请提供其他E-mail地址",
+  "error_api_email_exists_not": "Email不存在",
+  "error_api_email_format": "请提供一个有效的Email",
+  "error_api_email_required": "Email是必填的",
+  "error_api_email_used": "Email已经被使用",
+  "error_api_email_verified": "Email已被验证",
+  "error_api_phone_required": "手机号码是必填的",
```

Comment:
One key is missing here: `error_api_general` see in `en.json` https://github.com/steemit/faucet/blob/3085e897e0ebdc59cad457dca15cc0d1485472af/src/locales/en.json#L31
This key was probably added recently.